src/ui/components/DataTable/Header.tsx

```diff
@@ -102,14 +110,105 @@ const SuperCols = ({
 	);
 };
 
-export const getSortClassName = (sortBys: SortBy[], i: number) => {
+const SortableColumnHandle = SortableHandle(
+	(props: { isDragged: boolean; selected: boolean; children: ReactNode }) => {
+		return (
+			<span
+				className={classNames("d-inline-block border", {
+					"user-select-none": props.isDragged,
+					"table-secondary": props.selected,
+				})}
+				data-movable-handle
+				style={{
+					cursor: props.isDragged ? "grabbing" : "grab",
+					padding: "0 0.2rem",
+				}}
+			>
+				{props.children}
+			</span>
+		);
+	},
+);
+SortableColumnHandle.propTypes = {
+	isDragged: PropTypes.bool.isRequired,
+};
+
+const SortableColumn = SortableElement(
+	(props: {
+		isDragged: boolean;
+		selected: boolean;
+		col: Col;
+		sortBy: SortBy | undefined;
+		colIndex: number;
+		handleColClick: (b: MouseEvent, a: string) => void;
+	}) => {
+		let className;
+		if (props.col.sortSequence && props.col.sortSequence.length === 0) {
+			className = null;
+		} else {
+			className = getSortClassName(
+				props.sortBy ? [props.sortBy] : [],
+				props.col.key,
+			);
+		}
+
+		return (
```

Comment:
Moving the sort arrows to the foreground means columns get wider. Not sure if that is a good thing. Probably not, since some tables are already quite wide on moblie.

Could this go back to how it was before? Click sorts, drag reorders? The roster page has table handles that are separately clickable and draggable, so I think it's possible.